bees/discordbee/discordbee.go

```diff
@@ -73,7 +73,7 @@ func (mod *DiscordBee) isReady(s *discordgo.Session, event *discordgo.Ready) {
 	// Default status - once the persistance layer exists:
 	// TODO: Persist status, use it here.
 	mod.LogDebugf("isReady called")
-	s.UpdateStatus(0, "with bees")
+	s.UpdateGameStatus(0, "with bees")
```

Comment:
What's the difference between `UpdateStatus` and `UpdateGameStatus`?

Reply:
No technical difference, UpdateGameStatus is the new name of the function with the library discordgo in v0.23.1, and UpdateStatus was removed

Reply:
Understood, thanks for clarifying!